src/components/footer.jsx

```diff
@@ -18,15 +18,15 @@ function Footer() {
 
   const primaryLinks = links.primary;
   const primaryLinksLength = primaryLinks.length;
-  const primaryLinksLabels = t('footer.primaryLinks');
+  const primaryLinksLabels = t('navigation.dropdowns').map((element) => element.simpleLinks).flat().concat(t('navigation.parentLinks'));
```

Comment:
Cool! I had never seen the `.flat()` method before

Reply:
Yep, I saved so many lines of code and potential recursion on my end.